package.json

```diff
@@ -21,10 +21,10 @@
   },
   "homepage": "https://github.com/vuejs/eslint-plugin-vue#readme",
   "peerDependencies": {
-    "eslint": "^2.0.0 || ^3.0.0"
+    "eslint": "^2.0.0 || ^3.0.0 || ^4.0.0"
   },
   "dependencies": {
-    "eslint-plugin-html": "^2.0.0",
-    "eslint-plugin-react": "^6.9.0"
+    "eslint-plugin-html": "^2.0.0 || ^3.0.0",
```

Comment:
`eslint-plugin-html` v3 looks to have huge breaking changes which affect to several core rules. 
I think better that we should not include `eslint-plugin-html` v3 into this PR. (we are preparing 3.0.0 beta which does not use `eslint-plugin-html`. So we cannot major-release with this.)